Makefile

```diff
@@ -149,16 +153,39 @@ help:
 	@echo " kvasir                   -- make Kvasir, the C front end"
 	@echo " very-clean               -- remove (most) all generated files"
 	@echo " dyncomp-jdk              -- Make file java/dcomp_rt.jar"
-	@echo "Creating the Daikon distribution:"
+	@echo " quick-test               -- simple verification of system"
```

Comment:
Should this be next to the "junit" and "test" targets?  More generally, what order are the targets in?  Also explain that these are quick system tests; that isn't clear from the current description.